modules/nw-sriov-device-discovery.adoc

```diff
@@ -1,24 +1,28 @@
 // Module included in the following assemblies:
 //
-// * networking/multiple_networks/configuring-sr-iov.adoc
+// * networking/hardware_networks/about-sriov.adoc
```

Comment:
`modules/nw-sriov-device-discovery.adoc` is directly included in `networking/hardware_networks/configuring-sriov-device.adoc` file. The latter is included in `networking/hardware_networks/about-sriov.adoc`
Shall we change it to `networking/hardware_networks/configuring-sriov-device.adoc` ?

Reply:
@zshi-redhat, updated!